optaplanner-quarkus-integration/optaplanner-quarkus/drools-integration-test/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2020 Red Hat, Inc. and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
Seperate issue, but we need to get rid of this verbosity, especially in the application.properties etc for optaplanner-quickstarts. I see few quickstarts still doing a license on each file (they just put one in the root folder).
I'll start a optaplanner-dev conversation.